device/arista/x86_64-arista_7800r3a_36d2_lc/media_settings.json

```diff
@@ -2,8137 +2,9300 @@
+                    },
+                    "post3": {
+                        "lane0": "0xffffffff",
+                        "lane1": "0xfffffffd",
+                        "lane2": "0xfffffffd",
+                        "lane3": "0xfffffffd",
+                        "lane4": "0xfffffffd",
+                        "lane5": "0xfffffffd",
+                        "lane6": "0xffffffff",
+                        "lane7": "0xfffffffc"
+                    },
+                    "pre1": {
+                        "lane0": "0xfffffff0",
+
+                        "lane1": "0xfffffff0",
+                        "lane2": "0xfffffff0",
+                        "lane3": "0xffffffef",
+                        "lane4": "0xfffffff0",
+                        "lane5": "0xfffffff0",
+                        "lane6": "0xfffffff0",
+                        "lane7": "0xfffffff2"
+                    },
+                    "pre2": {
+                        "lane0": "0x2",
+                        "lane1": "0x3",
+                        "lane2": "0x3",
+                        "lane3": "0x2",
+                        "lane4": "0x2",
+                        "lane5": "0x3",
+                        "lane6": "0x2",
+                        "lane7": "0x2"
+                    }
                 }
             },
             "Default": {
-                "main": {
-                    "lane0": "0x59",
-                    "lane1": "0x55",
-                    "lane2": "0x50",
-                    "lane3": "0x4b"
-                },
-                "post1": {
-                    "lane0": "0xffffffe3",
-                    "lane1": "0xffffffe7",
-                    "lane2": "0xffffffe9",
-                    "lane3": "0xffffffeb"
-                },
-                "post2": {
-                    "lane0": "0x0",
-                    "lane1": "0x0",
-                    "lane2": "0x0",
-                    "lane3": "0x0"
-                },
-                "post3": {
-                    "lane0": "0x0",
-                    "lane1": "0x0",
-                    "lane2": "0x0",
-                    "lane3": "0x0"
-                },
-                "pre1": {
-                    "lane0": "0xfffffff8",
-                    "lane1": "0xfffffff9",
-                    "lane2": "0xfffffffb",
-                    "lane3": "0xfffffffc"
-                },
-                "pre2": {
-                    "lane0": "0x0",
-                    "lane1": "0x0",
-                    "lane2": "0x0",
-                    "lane3": "0x0"
+                "speed:CAUI-4|100GAUI-4|25G": {
+                    "main": {
+                        "lane0": "0x59",
+                        "lane1": "0x55",
+                        "lane2": "0x50",
+                        "lane3": "0x4b",
+                        "lane4": "0x0",
+                        "lane5": "0x0",
+                        "lane6": "0x0",
+                        "lane7": "0x0"
```

Comment:
@abohanyang please remove the extra lanes > 3 which are NOT applicable for 100G NRZ